surveyor/surveyor.go

```diff
@@ -336,11 +343,59 @@ func (s *Surveyor) startHTTP() error {
 	return nil
 }
 
+func (s *Surveyor) startObservations() error {
+	observationsGauge.Set(0)
+
+	dir := s.opts.ObservationConfigDir
+	if dir == "" {
+		log.Printf("Skipping observation startup, no directory configured")
+		return nil
+	}
+
+	fs, err := os.Stat(dir)
+	if err != nil {
+		return fmt.Errorf("could not start observations, %s does not exist", dir)
+	}
+
+	if !fs.IsDir() {
+		return fmt.Errorf("observations dir %s is not a directory", dir)
+	}
+
+	err = filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if filepath.Ext(info.Name()) != ".json" {
+			return nil
+		}
+
+		obs, err := NewServiceObservation(path, s.opts)
+		if err != nil {
+			return fmt.Errorf("could not create observation from %s: %s", path, err)
```

Comment:
do you think this is right @ColinSullivan1 ie. should failing to start a single observable drop the entire surveyor?

Reply:
imo that's the path of least surprise.  It'd force the user to see what went wrong, rather than have them wondering why they weren't getting data they intended to until they dig and look at logs.  Also, it'd be something we can relax later if users really don't like that behavior (vs the other way around...  if it's an warning now and we wanted to later make this an error it could break existing surveyor deployments).